packages/frontend/src/components/chart/configure/render/milestones/getMilestoneHtml.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { renderToStaticMarkup } from 'react-dom/server'
+
+import { MilestoneIcon } from '../../../../icons/symbols/MilestoneIcon'
+
+const ICON_SIZE = 20
+const OFFSET = ICON_SIZE / 2
+
+export function getMilestoneHtml(x: number) {
+  return renderToStaticMarkup(
+    <div className="absolute z-50" style={{ left: x - OFFSET, top: -OFFSET }}>
+      <MilestoneIcon
+        width={ICON_SIZE}
+        height={ICON_SIZE}
+        viewBox={`0 0 ${ICON_SIZE} ${ICON_SIZE}`}
+      />
+    </div>,
+  )
+}
```

Comment:
As much as I like the code clarity here this solution imports `react` as well as `react-dom/server` which weren't previously imported in frontend scripts. For this one edge case please use template strings.